pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -177,6 +177,10 @@ func (plugin *kubenetNetworkPlugin) Name() string {
 	return KubenetPluginName
 }
 
+func (plugin *kubenetNetworkPlugin) Capabilities() uint {
```

Comment:
instead of implementing this method over and over can you just embed a struct like:
```go
type pluginCapabilities uint

func (pc *pluginCapabilities) BWShaping() bool {
  return pc & NET_PLUGIN_CAPABILITY_SHAPING
}

 type kubenetNetworkPlugin struct {
  pluginCapabilities
 }
 
 func NewPlugin() network.NetworkPlugin {
  return &kubenetNetworkPlugin{pluginCapabilities: 0}
 }
```